modules/blocks.php

```diff
@@ -386,3 +386,38 @@ class="components-text-control__input wp-block-jetpack-mailchimp-email"
 	$html = ob_get_clean();
 	return $html;
 }
+
+/**
+ * Visited Block
+ */
+jetpack_register_block( 
```

Comment:
This will need reworking since `jetpack_register_block` has been deprecated as of https://github.com/Automattic/jetpack/pull/11212 (cc @ockham)

See example how e.g. simple payments now registers :https://github.com/Automattic/jetpack/blob/ae23942fd76ca8ff56130ccf234cf891e6536386/modules/simple-payments/simple-payments.php#L63-L73

(it's important to do `function_exists( 'register_block_type' )` now)